client/app/scripts/superdesk-ingest/views/settings/ingest-routing-schedule.html

```diff
@@ -6,13 +6,55 @@
     </div>
 
     <div class="field">
-        <label translate>Start Time</label>
-        <span sd-timepicker-alt data-model="rule.schedule.hour_of_day_from"></span>
+        <label translate>Start Time (inclusive)</label>
+        <span
+            sd-timepicker-alt
+            no-utc-convert="true"
+            data-model="rule.schedule.hour_of_day_from"></span>
     </div>
 
     <div class="field">
-        <label translate>End Time</label>
-        <span sd-timepicker-alt data-model="rule.schedule.hour_of_day_to"></span>
+        <label translate>End Time (exclusive)</label>
+        <span
+            sd-timepicker-alt
+            no-utc-convert="true"
+            data-model="rule.schedule.hour_of_day_to"></span>
+    </div>
+
+    <div class="field">
+        <label translate>Time Zone</label>
+        <div class="control" ng-hide="rule.schedule.time_zone">
+            <ul sd-typeahead
+                items="matchingTimeZones"
+                term="tzSearchTerm"
+                search="searchTimeZones(term)"
+                select="selectTimeZone(item)"
+                always-visible="tzSearchTerm">
+                <li
+                    typeahead-item="tz"
+                    ng-repeat="tz in matchingTimeZones track by tz"
+                    >{{ ::tz }}</li>
+                <li ng-show="tzSearchTerm && matchingTimeZones.length === 0"
```

Comment:
I found it quite user-unfriendly when you type something and get no results, it almost seems like the widget is not working because of the lack of visual feedback. I can remove it for the sake of consistency, but would personally recommend adding it to other places as well.

Of course it's the UX specialists who have the final word on this, please let us know what you think, thanks! :)